aten/src/ATen/Context.cpp

```diff
@@ -145,7 +145,9 @@ void Context::setAllowTF32OneDNN(bool b){
 #ifdef USE_XPU
   allow_tf32_onednn = b;
 #else
-  TORCH_WARN("TF32 acceleration on top of oneDNN is available for Intel GPUs. The current Torch version does not have Intel GPU Support.");
+  if(b){
```

Comment:
```suggestion
  if (b) {
```

Reply:
Please fix the lint.

Reply:
Why is the linter issue not captured?

Reply:
> Please fix the lint.
I wrongly thought you use some git button to fix the lint issue since you resolve the comment... hah, I will fix them, thanks for reminding.

> Why is the lint issue not captured?

@EikanWang this file is not in `.lintrunner.toml` .  (You may see other `if(b)` like case in this file) . Maybe this is a choice of other maintainers...

Reply:
fixed in the new commit

Reply:
No. Refer to https://github.com/pytorch/pytorch/blob/a8d0c5c92818186119d4a94d98999acc3f549a7e/.lintrunner.toml#L55-L57, all .cpp file in aten/src/ATen are excluded from clangformat.

Reply:
> > Please fix the lint.
> 
> I wrongly thought you use some git button to fix the lint issue since you resolve the comment... hah, I will fix them, thanks for reminding.

Haha, just a reminder. I will break the ghstack merge label if I use the git button. So I revert my patch.